nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3533,6 +3537,7 @@ async fn builtin_users_get_user(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+// TODO is this the right interface?
```

Comment:
a) I don't think so, b) probably not. I believe @iliana is actively working on this

Reply:
I don't think this is the right abstraction and I wouldn't want to ship a production rack with this endpoint, personally. (I'm currently writing an RFD on the topic of metrics that are associated with resources we already have endpoints for)

Reply:
@iliana I was unaware of the work you were doing here: glad you're taking this on!

Should I move this into another tag or something to clearly denote that it's a temporary interface? Or should I just ignore it?

Reply:
I'd put a TODO about deleting it, but otherwise ignore it. There are some tests related to it, and possibly other things I've forgotten about, that would make just deleting it fail.

Reply:
(Maybe also a GH issue so we can remember it there, too.)

Reply:
https://github.com/oxidecomputer/omicron/issues/1331